src/reducers/ui.js

```diff
@@ -76,6 +78,8 @@ export const DEFAULT_UI = {
     },
     itemsByDimension: {
         [DIMENSION_ID_ORGUNIT]: [],
+        [DIMENSION_TYPE_EVENT_STATUS]: [], // XXX should this default be set in the component instead? since these are fixed dimensions perhaps this default here makes sense
```

Comment:
remember to remove comment. I personally like having defaults in redux so the components don't have to worry about it. But not too opinionated on that.

Reply:
We can leave it there then and remove the comment.
```suggestion
        [DIMENSION_TYPE_EVENT_STATUS]: [],
```